spec/models/manageiq/providers/ibm_power_vc/cloud_manager/refresher_spec.rb

```diff
@@ -95,6 +96,15 @@ def assert_specific_vm
       )
     end
 
+    def assert_specific_operating_system
+      vm = ems.vms.find_by(:ems_ref => instid)
+      os = vm.operating_system
+      expect(os).to have_attributes(
+        :product_name => "linux_redhat",
+        :version      => "Linux/CentOS Stream 4.18.0-408.el8.ppc64le 8"
```

Comment:
Hm I don't think the entire OS name should be in version, probably just `4.18.0-408.el8.ppc64le`

Reply:
I can't recall what the other VMs have, but I recall one of these fields had all kinds of text in it.

Reply:
Ideally I could parse this into something like:
```
:product_name   => "linux_redhat",
:distro         => "Linux/CentOS Stream",
:version        => "8",
:kernel_release => "4.18.0-408.el8.ppc64le",
:description    => "Linux/CentOS Stream 4.18.0-408.el8.ppc64le 8", 
```
and I guess I could if I updated the `operating_systems` table schema to include `kernal_release` and `description`. Instead I took this shortcut and jammed the whole value into `version` where at least all of the info is available.

In trying to use the existing schema, I'm a bit confused at how the UI displays data. To illustrate, I updated all empty character columns of an entry with placeholder values and took a look at the UI:
```sql
localhost:5432 root@master=# select * from operating_systems where id = 58;
-[ RECORD 1 ]---------+---------------------------------------------
id                    | 58
name                  | name
product_name          | linux_redhat
version               | Linux/CentOS Stream 5.14.0-404.el9.ppc64le 9
build_number          | build_number
system_root           | system_root
distribution          | rhel
product_type          | product_type
service_pack          | service_pack
productid             | productid
vm_or_template_id     | 58
host_id               | 
bitness               | 42
product_key           | 
pw_hist               | 
max_pw_age            | 
min_pw_age            | 
min_pw_len            | 
pw_complex            | 
pw_encrypt            | 
lockout_threshold     | 
lockout_duration      | 
reset_lockout_counter | 
system_type           | system_type
computer_system_id    | 
kernel_version        | 
```
![image](https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/assets/1637291/54283940-d164-4781-8289-485a70c381cf)

Notably:
- `name`, `system_root`, `distribution`, `product_type`, `service_pack` and `system_type` aren't displayed at all.
- `bitness` is displayed as "System Type"

Reply:
Here's the UI textual OS info helper:
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/textual_mixins/textual_os_info.rb#L26-L37

Reply:
There is an `OperatingSystem#name` column that I could see showing the full name

Reply:
> There is an `OperatingSystem#name` column that I could see showing the full name

Yeah, that would work for reporting as-is. If we go that route I'd also like to update the [textual_os_info module](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/textual_mixins/textual_os_info.rb#L26-L37) to display `name`.

Reply:
See also https://github.com/ManageIQ/manageiq/blob/master/app/models/operating_system.rb#L77-L132.  Note that we process a lot of the other fields by using this convoluted algorithm.

Reply:
I pushed 0c707bc to move the value into the `name` column. As far as I can tell, there aren't any existing provides that set a `name` value.

Reply:
I also meant to mention that I opened a related PR to display the `name` value as "Description" in the UI:
https://github.com/ManageIQ/manageiq-ui-classic/pull/9110